### PR TITLE
Containers must be ready as well as running to show as running

### PR DESCRIPTION
**Pull request by kipz**

### CODE REVIEW

File: lib/handlers/event/push/rendering/PushNodeRenderers.ts [hunk 1]
```diff
@@ -505,7 +505,7 @@ export class K8PodNodeRenderer extends AbstractIdentifiableContribution
                     envs.push(env);
                 }
                 pod.containers.forEach(c => {
-                    if (c.state === "running") {
+                    if (c.state === "running" && c.ready === true) {
```
Comment (ddgenome) [inline lib/handlers/event/push/rendering/PushNodeRenderers.ts:508]: I guess there is no "ready" state?

**Review by ddgenome — COMMENTED** [review]
One question.

File: lib/handlers/event/push/rendering/PushNodeRenderers.ts (hunk 1 above)
Comment (kipz) [reply lib/handlers/event/push/rendering/PushNodeRenderers.ts:508]: Seems not. Something can be running but not ready - that's a k8s thing.

Comment (ddgenome) [reply lib/handlers/event/push/rendering/PushNodeRenderers.ts:508]: Right, but not an Atomist thing. Seems reasonable if we are only reporting running then we should add this check to combine Kubernetes ideas of ready and running.

**Review by ddgenome — APPROVED** [review]
LGTM

**Comment by cdupuis** [comment]
Pull request was successfully rebased onto `30c83e9` by @cdupuis:
- a2a9a87 _Apply target for SDM packages_
- 30c83e9 _Merge pull request <https://github.com/atomist/sdm-pack-lifecycle/issues/19|#19> from atomist/sdm-pack-lifec..._

**Comment by cdupuis** [comment]
Pull request was successfully rebased onto `9ebe46e` by @cdupuis:
- 9ebe46e _Autofix: Third party licenses_

**Review by cdupuis — APPROVED** [review]

**Comment by cdupuis** [comment]
Pull request auto merged by Atomist.

* 2 approved reviews by @ddgenome, @cdupuis
* 2 successful checks

[atomist:generated] [auto-merge:on-approve]

**Pull request merged**